tasks/prelim.yml

```diff
@@ -8,6 +8,19 @@
   tags:
       - rule_1.1.1.2
       - always
+  when:
+      - ubtu22cis_rule_1_1_1_2
+
+- name: "PRELIM | Register if squashfs is built into the kernel"
+  ansible.builtin.shell: more /lib/modules/$(uname -r)/modules.builtin | grep  -c "squashfs"
```

Comment:
FYI: This is causing things to hang when running via Packer/Ansible. Likely due to `more` instead of say `cat`

Reply:
@tomi-bigpi:  This PR has already been merged ... could you open an issue about this behavior? Sorry, that I broke something :(